WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -177,12 +178,16 @@ public static void addMedia(Activity activity) {
     }
 
     public static void viewMyProfile(Context context) {
+        AccountHelper.getDefaultAccount().fetchAccountSettings();
+
         Intent intent = new Intent(context, MyProfileActivity.class);
         AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.OPENED_MY_PROFILE);
         slideInFromRight(context, intent);
     }
 
     public static void viewAccountSettings(Activity activity) {
+        AccountHelper.getDefaultAccount().fetchAccountSettings();
```

Comment:
It's seems that this is more like a matter of preference and judging by the low usage count of these functions in the current codebase/PR I think it doesn't warrant blocking the PR on it.